File: submissions/sm_037_srikanth/week_03/day_3/session_2/split_string.js
var str = "Masai School Bangalore";
var splitChar ='a';

function split(str, splitChar){

    var array =[];
    var char='';
    for(i=0;i<str.length;i++){
        if(str[i] != splitChar){
          char= char + str[i];
        }
        else{
            array.push(char);
            char='';
        }
    }
    array.push(char);
    return console.log(array);
}

split(str,splitChar);

File: submissions/sm_021_piyush-jain/week_13/day_4/split_string.py
str="a,b"
splitChar=[","]
x=0
data=[]
temp=""
while x<len(str):
    if(str[x] not in splitChar):
        temp=temp+str[x]
        x=x+1
    else:
        print(temp)
        data.append(temp)
        temp=""
        x=x+1 

if(x==len(str)):
    data.append(temp) 
      
print(data)
print(str.split(","))


